### DISCUSSION TITLE
SharePoint not showing translations after manual import

### DISCUSSION THREAD

**Discussion by jul-dum** [dotnet/ResXResourceManager#557] [Q&A]
Something's not working as I expected and I'm not sure where I went wrong. I'm using SharePoint in M365 and the standalone version of ResX Resource Manager.

My SharePoint site's default language is set to "French". I've added "English" as a secondary language.

1. I used SharePoint's Export translations function to export the French text. This downloads a ResX file named "[site_name].fr-FR.resx" in my Downloads folder.
2. I opened ResX Resource Manager and set my neutral language as French (France) (fr-FR). I set the source to my Downloads folder.
3. I added a column for English (United States) and started translating. This created a new file named "[site_name].fr-FR.en-US.resx" in my Downloads folder.
4. I used SharePoint's Import translations function to import the en-US resx file. I didn't get an error message.

Unfortunately, when I ask a colleague's whose regional language settings are set to English to check if the translation is showing, she's still seeing my site (mainly, a specific list) in French. Yet, that list was included in the imported translations.

Am I using the tool correctly?

**Comment by tom-englert** [comment]
You must follow the naming convention of .resx files:
Neutral: `[site_name].resx`
French: `[site_name].fr-FR.resx`
English: `[site_name].en-US.resx`

**Reply by jul-dum** [reply]
This was a valuable tip and the interface made a lot more sense, but unfortunately, I'm still not seeing my translations once I upload the en-US.resx file to my SharePoint site  :(

**Reply by tom-englert** [reply]
Well, that's a question for SharePoint, not for ResX Resource Manager
